ftplugin/dirvish.vim

```diff
@@ -5,6 +5,8 @@ if !hasmapto('<Plug>(dirvish_quit)', 'n')
 endif
 
 nnoremap <buffer><silent> <Plug>(dirvish_up) :<C-U>exe "Dirvish %:h".repeat(":h",v:count1)<CR>
```

Comment:
The reason this is defined here and in `plugin/dirvish.vim` is because it's actually slightly different: `Dirvish %:h` vs `Dirvish %:p`.

Since the global mappings you've added aren't different for buffer-local operation, I think we only need them in `plugin/dirvish.vim`.